vehicles/src/modules/loa/entities/loa-permit-type-details.entity.ts

```diff
@@ -0,0 +1,45 @@
+import { ApiProperty } from '@nestjs/swagger';
+import {
+  Entity,
+  Column,
+  PrimaryGeneratedColumn,
+  JoinColumn,
+  ManyToOne,
+} from 'typeorm';
+import { AutoMap } from '@automapper/classes';
+import { Base } from 'src/modules/common/entities/base.entity';
+import { PermitType } from 'src/common/enum/permit-type.enum';
+import { LoaDetail } from './loa-detail.entity';
+
+@Entity({ name: 'permit.ORBC_LOA_PERMIT_TYPE_DETAILS' })
+export class LoaPermitType extends Base {
+  @AutoMap()
+  @ApiProperty({
+    example: '1',
+    description: 'Unique identifier for the loa permit type.',
+  })
+  @PrimaryGeneratedColumn({ type: 'bigint', name: 'LOA_PERMIT_TYPE_ID' })
+  id: string;
+
+  @AutoMap(() => LoaDetail)
+  @ManyToOne(() => LoaDetail, (LoaDetail) => LoaDetail.loaPermitTypes, {
+    orphanedRowAction: 'delete',
```

Comment:
Are we not supposed to soft-delete in OnRouteBC as advised by PM? Please confirm with JF about the design